### PR TITLE
Add License info on all supported datasets

### PR DESCRIPTION
**Pull request by mpariente**
I'm going to ask the owners of the License / creators of the dataset to approve the License infos, directly in the code. Sorry for pinging.

These license info will be used to automatically generate the Attribution statement in the license of pretrained models.

### CODE REVIEW

File: asteroid/data/dns_dataset.py [hunk 1]
```diff
@@ -34,3 +34,26 @@ def __getitem__(self, idx):
...
+
+dns_license = dict(
+    title='Deep Noise Suppression (DNS) Challenge',
+    title_link='https://github.com/microsoft/DNS-Challenge',
+    author='Microsoft',
+    author_link='https://www.microsoft.com/fr-fr/',
+    license='CC BY-NC 4.0',
+    license_link='https://creativecommons.org/licenses/by-nc/4.0/',
+    non_commercial=False,
+)
```
Comment (mpariente) [inline asteroid/data/dns_dataset.py:59]: @chandanka90 could you please review that these informations are correct please?

File: asteroid/data/librimix_dataset.py [hunk 2]
```diff
@@ -101,3 +102,34 @@ def __getitem__(self, idx):
...
+
+librispeech_license = dict(
+    title='LibriSpeech ASR corpus',
+    title_link='http://www.openslr.org/12',
+    author='Open SLR',
+    author_link='http://www.openslr.org/',
+    license='CC BY 4.0',
+    license_link='https://creativecommons.org/licenses/by/4.0/',
+    non_commercial=False
+)
```
Comment (mpariente) [inline asteroid/data/librimix_dataset.py:135]: @vdp & @danpovey could you please review that these informations are correct please?

File: asteroid/data/musdb18_dataset.py [hunk 3]
```diff
@@ -213,3 +213,26 @@ def get_tracks(self):
...
+
+musdb_license = dict(
+    title='MUSDB18',
+    title_link='https://sigsep.github.io/datasets/musdb.html',
+    author='SigSep',
+    author_link='',
+    license='CC BY-NC-SA 4.0',
+    license_link='https://creativecommons.org/licenses/by-nc-sa/4.0/',
+    non_commercial=True,
+)
```
Comment (mpariente) [inline asteroid/data/musdb18_dataset.py:238]: @faroit & @aliutkus could you please review that these informations are correct?

**Comment by mpariente** [comment]
[Test are failing because Zenodo is down]

File: asteroid/data/librimix_dataset.py (hunk 2 above)
Comment (danpovey) [reply asteroid/data/librimix_dataset.py:135]: Looks right to me, except you can perhaps put 'Vassil Panayotov' as `author`.. OpenSLR is just the release mechanism.

Comment (mpariente) [reply asteroid/data/librimix_dataset.py:135]:
```suggestion
librispeech_license = dict(
    title='LibriSpeech ASR corpus',
    title_link='http://www.openslr.org/12',
    author='Vassil Panayotov',
    author_link='https://github.com/vdp',
    license='CC BY 4.0',
    license_link='https://creativecommons.org/licenses/by/4.0/',
    non_commercial=False
)
```
Should I add  '& Daniel Povey'?

**Comment by faroit** [comment]
Musdb18 is (unfortunately) not CC

File: asteroid/data/librimix_dataset.py (hunk 2 above)
Comment (danpovey) [reply asteroid/data/librimix_dataset.py:135]: No its OK,  Vassil did it all himself..

**Comment by mpariente** [comment]
> Musdb18 is (unfortunately) not CC

I didn't find the info about the license in the SigSep page, only info about the compsition (and their license).

**Comment by mpariente** [comment]
I'll merge this to go forward with model sharing but feel free to comment or open a PR if the info is not the right one.

File: asteroid/data/musdb18_dataset.py (hunk 3 above)
Comment (mpariente) [reply asteroid/data/musdb18_dataset.py:238]:
```suggestion
musdb_license = dict(
)
```
For now, I'll keep it like this and we'll get to it when we need to share asteroid models from MUSDB

**Pull request merged**